CMakeLists.txt

```diff
@@ -169,17 +169,20 @@ if ("${CMAKE_CXX_COMPILER_ID}" MATCHES "GNU")
 endif ("${CMAKE_CXX_COMPILER_ID}" MATCHES "GNU")
 
 if (APPLE)
-   set(CMAKE_CXX_FLAGS_DEBUG   "-std=c++11 -fPIC -stdlib=libc++ -g -Wno-inconsistent-missing-override")
-   set(CMAKE_CXX_FLAGS_RELEASE "-std=c++11 -fPIC -stdlib=libc++ -O2 -DNDEBUG -DQT_NO_DEBUG -Wno-inconsistent-missing-override")
+   set(CMAKE_CXX_FLAGS         "${CMAKE_CXX_FLAGS} -std=c++11 -fPIC -stdlib=libc++ -Wno-inconsistent-missing-override")
+   set(CMAKE_CXX_FLAGS_DEBUG   "-g")
+   set(CMAKE_CXX_FLAGS_RELEASE "-O2 -DNDEBUG -DQT_NO_DEBUG")
 else (APPLE)
    if (MINGW)
       # -mno-ms-bitfields see #22048
-      set(CMAKE_CXX_FLAGS_DEBUG   "-std=c++11 -mno-ms-bitfields -g")
-      set(CMAKE_CXX_FLAGS_RELEASE "-std=c++11 -mno-ms-bitfields -O2 -DNDEBUG -DQT_NO_DEBUG")
+      set(CMAKE_CXX_FLAGS         "${CMAKE_CXX_FLAGS} -std=c++11 -mno-ms-bitfields")
+      set(CMAKE_CXX_FLAGS_DEBUG   "-g")
```

Comment:
No, that would be wrong, because the *compile* macros of cmake *all already use* both, and you’re even supposed to. (I actually researched that quite long before submitting.)

Additionally, the original cause of this change was that Debhelper uses neither DEBUG nor RELEASE build but NONE, for various reasons.

Having things in the proper places (`CMAKE_CXX_FLAGS` for everything that is not DEBUG or RELEASE-specific) is the cmake way to fix things, and the correct one.